package.json

```diff
@@ -1,73 +1,71 @@
 {
-  "name": "mpc-phase2-suite",
-  "description": "MPC Phase 2 suite of tools for conducting zkSNARKs trusted setup ceremonies",
-  "repository": "git@github.com:quadratic-funding/mpc-phase2-suite.git",
-  "homepage": "https://github.com/quadratic-funding/mpc-phase2-suite",
-  "bugs": "https://github.com/quadratic-funding/mpc-phase2-suite/issues",
-  "license": "MIT",
-  "private": true,
-  "workspaces": {
-    "packages": [
-      "phase2cli",
-      "firebase"
+    "name": "zkmpc",
+    "description": "MPC Phase 2 suite of tools for conducting zkSNARKs trusted setup ceremonies",
+    "repository": "git@github.com:quadratic-funding/mpc-phase2-suite.git",
+    "homepage": "https://github.com/quadratic-funding/mpc-phase2-suite",
+    "bugs": "https://github.com/quadratic-funding/mpc-phase2-suite/issues",
+    "license": "MIT",
+    "private": true,
+    "keywords": [
+        "typescript",
+        "zero-knowledge",
+        "zk-snarks",
+        "phase-2",
+        "trusted-setup",
+        "ceremony",
+        "snarkjs",
+        "circom"
     ],
-    "nohoist": [
-      "**/snarkjs",
-      "**/snarkjs/**"
+    "scripts": {
+        "build": "lerna run build",
+        "pretest": "yarn build",
+        "test": "export GOOGLE_APPLICATION_CREDENTIALS=\"./packages/backend/serviceAccountKey.json\" && jest --config=jest.json --detectOpenHandles --forceExit  --coverage",
+        "test:watch": "export GOOGLE_APPLICATION_CREDENTIALS=\"./packages/backend/serviceAccountKey.json\" && jest --config=jest.json --watch --detectOpenHandles --forceExit  --coverage",
```

Comment:
Seems like that in watch mode `--detectOpenHandles` and `--forceExit` are not needed

Reply:
I got the following warning when removing the `--detectOpenHandles`

`A worker process has failed to exit gracefully and has been force exited. This is likely caused by tests leaking due to improper teardown. Try running with --detectOpenHandles to find leaks. Active timers can also cause this, ensure that .unref() was called on them.`

Reply:
Even for `test:watch`?

Reply:
Yes, that's the one I executed rn!

Reply:
My bad here, ignore this comment please

Reply:
No worries <3